packages/scheduled-payments-client/ember-cli-build.js

```diff
@@ -6,11 +6,73 @@ process.env.EMBROIDER_REBUILD_ADDONS = '@cardstack/boxel';
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const { Webpack } = require('@embroider/webpack');
 const { compatBuild } = require('@embroider/compat');
+const svgoUniqueId = require('svgo-unique-id');
 
 module.exports = function (defaults) {
   let app = new EmberApp(defaults, {
-    // Add options here
+    svgJar: {
+      strategy: 'inline',
+      sourceDirs: ['public'],
+      optimizer: {
+        svgoModule: require('svgo'),
+        plugins: [
+          { removeTitle: false },
+          { removeDesc: { removeAny: false } },
+          { removeViewBox: false },
+          {
+            cleanupIDs: { minify: false },
+          },
+          {
+            prefixIds: false,
+          },
+          {
+            uniqueID: svgoUniqueId,
+          },
+        ],
+      },
+    },
```

Comment:
I've tried to add most of the boxel stuff, but the requirements were not clear, maybe we should think about documenting it, or create some sort of template.

Reply:
Agreed. It is pretty esoteric right now, mostly with the webpack processing requirements.